module/VuFind/src/VuFind/Recommend/ConsortialVuFind.php

```diff
@@ -208,7 +212,7 @@ public function process($results)
      */
     public function getResults()
     {
-        if (!$this->hasMinimumConfig) {
+        if (!$this->hasMinimumConfig || !$this->queryString) {
```

Comment:
Just to be a little more strict, I think it would be helpful to change the type of the queryString property to `?string` and initialize it to null so we're not potentially accessing an uninitialized variable here.

Reply:
Initializing to null makes sense.